media/static-bundles.json

```diff
@@ -1483,10 +1489,18 @@
       "files": [
         "protocol/js/protocol-modal.js",
         "js/base/mozilla-lazy-load.js",
-        "js/foundation/annual-report-2019.es6.js"
+        "js/foundation/annual-report-2019-2020.es6.js"
       ],
       "name": "annual_report_2019"
     },
+    {
+      "files": [
+        "protocol/js/protocol-modal.js",
+        "js/base/mozilla-lazy-load.js",
+        "js/foundation/annual-report-2019-2020.es6.js"
+      ],
+      "name": "annual_report_2020"
```

Comment:
Same as the CSS bundle, we don't really need two JS bundles if they have all the same files in them.